domains/_vercel.abhirupkumar.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "abhirupkumar",
+    "email": "abhirupkumar2003@gmail.com",
+    "discord": "1015889214904205362"
+  },
+  "record": {
+    "TXT": ["vc-domain-verify=abhirupkumar.is-a.dev,0ccc6a52e7129f6028ec", "vc-domain-verify=akb.is-a.dev,0077abfff592aad640d6"]
```

Comment:
you should put `vc-domain-verify=akb.is-a.dev,0077abfff592aad640d6` in your `_vercel.akb.is-a.dev`, not on this file
```suggestion
    "TXT": ["vc-domain-verify=abhirupkumar.is-a.dev,0ccc6a52e7129f6028ec"]
```